pom.xml

```diff
@@ -143,18 +143,19 @@
                 <artifactId>exec-maven-plugin</artifactId>
                 <executions>
                     <execution>
-                      <phase>verify</phase>
-                      <goals>
-                        <goal>exec</goal>
-                      </goals>
+                       <id>create-sevntu-patch</id>
```

Comment:
Like we said in the issue, it isn't really sevntu creating this, and this isn't used by sevntu but [patch-filters](https://github.com/checkstyle/patch-filters/blob/master/pom.xml#L13).

Since this is created for the whole repo, maybe it is better to call it something like `create-git-patch-file`?